runtime_scan/pkg/orchestrator/config.go

```diff
@@ -73,12 +75,19 @@ const (
 
 	DefaultTrivyServerTimeout = 5 * time.Minute
 	DefaultGrypeServerTimeout = 2 * time.Minute
+
+	DefaultControllerStartupDelay = 15 * time.Second
 )
 
 type Config struct {
 	AWSConfig             *aws.Config
 	ScannerBackendAddress string
 
+	// The Orchestrator starts the Controller(s) in a sequence and the ControllerStartupDelay is used for waiting
```

Comment:
Saw that it was merged but maybe worth stating in the comment that the delay is between controller starts (just for users not to be confused why the first one starts without a delay). "..used for waiting between Controller starts to ..."